container.go

```diff
@@ -242,3 +249,9 @@ func (c *ContainerRequest) validateMounts() error {
 	}
 	return nil
 }
+
+// WithFile set the file to be copied before starting a created container
+func (c ContainerRequest) WithFile(f ContainerFile) ContainerRequest {
```

Comment:
I think this method is exposed but not used. In the tests, the Files array is directly initialised, which seems easier to me that adding this method to the API